engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -263,22 +277,52 @@ def has_no_regcode?(auth_header)
 
         protected
 
+        # rubocop:disable Metrics/PerceivedComplexity
+        # rubocop:disable Metrics/CyclomaticComplexity
         def scc_activate_product
           logger.info "Activating product #{@product.product_string} to SCC"
-          auth = request.headers['HTTP_AUTHORIZATION']
+          auth = nil
+          auth = request.headers['HTTP_AUTHORIZATION'] if request.headers.include?('HTTP_AUTHORIZATION')
```

Comment:
What does `request.headers['HTTP_AUTHORIZATION']` evaluate to if it's not included in the headers? It should be `nil` anyway, in which case you can just shorten L284 & 285 to 

```
auth = request.headers['HTTP_AUTHORIZATION']
```